src/components/GraphWorkshop/ParametersPanel/ParametersPanel.tsx

```diff
@@ -107,7 +148,7 @@ const ParametersPanel = () => {
 					<RenderIf condition={yAxis?.value === Aggragation.uniqueValues}>
 						<Dropdown
 							value={yAxisField}
-							onChange={setYAxisField}
+							onChange={(yAxisField) => {setYAxisField(yAxisField); setDataFields(null)}}

```

Comment:
create common function that get a callback (setYAxisField) and does setDataFields(null) by itself.
use it in those 3 places

Reply:
wouldn't it be a very specific function? how should I call it

Reply:
overrideFieldBy(callback)  maybe?